integration-tests/src/test/java/com/datastax/oss/driver/osgi/OsgiCustomLoadBalancingPolicyIT.java

```diff
@@ -27,9 +33,10 @@
  * DynamicImport-Package: *</code>.
  */
 public class OsgiCustomLoadBalancingPolicyIT extends OsgiBaseIT {
-  @Override
-  public Option[] additionalOptions() {
-    return new Option[0];
+
+  @Configuration
+  public Option[] config() {
```

Comment:
I know that these things are a matter of taste, but I personally like it better when each class declares its own config, it makes it easier for me to understand what bundles are going to be provisioned. Hope everybody will agree here.